src/hotspot/cpu/x86/stubGenerator_x86_64_fmod.cpp

```diff
@@ -480,9 +470,9 @@ address StubGenerator::generate_libmFmod() {
     //   while (bs <= a)
     __ ucomisd(xmm0, xmm1);
     __ movapd(xmm4, xmm0);
-    __ jcc(Assembler::aboveEqual, L_117f);
-    __ jmp(L_11af);
-    __ align32();
+    __ jccb(Assembler::aboveEqual, L_117f);
```

Comment:
This is a bunded label jump, Assembler::jcc should automatically optimize it with short jump encoding, but its not a blocker.